packages/core/crates/core-transport/src/lib.rs

```diff
@@ -52,11 +52,16 @@ use libp2p::request_response::{RequestId, ResponseChannel};
 use std::pin::Pin;
 use std::sync::Arc;
 use utils_log::{error, info};
-use utils_types::primitives::Address;
+use utils_types::primitives::{Address, U256};
 
 use core_types::acknowledgement::AcknowledgedTicket;
 use core_types::channels::ChannelEntry;
 
+lazy_static::lazy_static! {
+    /// Fixed price per packet to 0.01 HOPR
+    static ref DEFAULT_PRICE_PER_PACKET: U256 = 10000000000000000u128.into();
+}
+
```

Comment:
It makes little sense to me, why the transport layer mechanism would hold the ticket price. The value should be somewhere else, like types or primitives.